ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.11.yaml

```diff
@@ -0,0 +1,198 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
```

Comment:
can we use `12`? just have a try. ocp 4.8 is very old.

Reply:
Done.